docs/contributing/developing.md

```diff
@@ -46,6 +46,18 @@ npm run build
 
 Any Wagtail sites you start up in this virtualenv will now run against this development instance of Wagtail. We recommend using the [Wagtail Bakery demo site](https://github.com/wagtail/bakerydemo/) as a basis for developing Wagtail. Keep in mind that the setup steps for a Wagtail site may include installing a release version of Wagtail, which will override the development version you've just set up. In this case, you should install the site before running the `pip install -e` step, or re-run that step after the site is installed.
 
+## Development on Windows
+Wagtail continues to update our documentation but development on Windows is not as documented as we would like it to be. We recommend setting up on a local virtual machine using our already available scripts, [docker-wagtail-develop](https://github.com/wagtail/docker-wagtail-develop) or [vagrant-wagtail-develop](https://github.com/wagtail/vagrant-wagtail-develop)
+
+If you are happy working on Windows, you will need to have the `make` utility installed in order to run commands associated with it. You can use Windows Powershell to install a package manager like [Chocolatey](https://chocolatey.org/install) and in turn use it install [make](https://chocolatey.org/install).
```

Comment:
```suggestion
If you are confident with Python and Node development on Windows and wish to proceed here are some helpful tips.

We recommend [Chocolatey](https://chocolatey.org/install) for managing packages in Windows. Once Chocolatey is installed you can then install the  [`make`](https://community.chocolatey.org/packages/make) utility in order to run common build and development commands.
```

Just trying to re-work this a bit to be less dense but also take you through discrete steps. Installing Chocolatey first and then make second.